src/main/java/solid/model/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,24 @@
+package solid.model;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    private static final int indexOperation = 0;
+    private static final int indexNameFruit = 1;
+    private static final int indexQuantity = 2;
+    private static final String splitRegexCsv = ",";
```

Comment:
All constants should follow THIS_UPPER_CASE naming. I mean all letters are uppercase